Components/Card/card.css

```diff
@@ -45,12 +47,44 @@ section .example-div {
     background-color: var(--white);
     width: 50%;
 }
-.card-image{
+
+.card-image {
     width: 100%;
 }
-.horizontal-card-image{
+
+.horizontal-card {
+    display: flex;
+    width: 40rem;
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
margin: 0 auto; works the same (1 line code instead of 2)

Reply:
Sure will consider this and update 👍🏻